### PR TITLE
Add info for learners about discussion notifications

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3815](https://openedx.atlassian.net/browse/DOC-3815)

Add info for learners about discussion notifications, including info about how to unsubscribe.

### Date Needed (optional)

13 November 2017

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @sanfordstudent, @iloveagent57, or @ssemenova 
- [x] Doc team review (copy edit): @edx/doc
- [ ] Product review: @shamck 
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 1]
```diff
@@ -58,17 +58,54 @@ number.
...
+
+==============================
+Receiving Email Notifications
+==============================
+
+By default, the first time another learner or member of the course team
+responds to a post that you have made, you receive an email notification.
+
+If you do not want to receive this email notification, clear the **Receive
+updates** check box before you add your post.
```
Comment (debchatigny) [inline en_us/shared/students/sfd_discussions/keep_up.rst]: Change check box to checkbox (GUI convention usage is one word).

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 2]
```diff
@@ -58,17 +58,54 @@ number.
...
+
+You only receive an email notification for the first response. For additional
+responses and comments, you receive information in a daily digest email
```
Comment (debchatigny) [inline en_us/shared/students/sfd_discussions/keep_up.rst]: Do you automatically get the daily digest mail, or do you have to subscribe? If the latter, I'd reword to clarify: 

You only receive an email notification for the first response. 
You can sign up to receive a daily digest email to be alerted of additional activity on your posts and posts that you are following. 

(hyperlink to 'receive a daily digest email')

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 3]
```diff
@@ -58,17 +58,54 @@ number.
...
+the daily digest, follow these steps.
 
-You have the option to receive an email message each day that summarizes
-discussion activity for the posts you are following. To receive this daily
-digest, select **Discussion** to go to the discussions home page, and then
-select the **Receive updates** check box in the **How to use edX discussions**
-graphic.
+#. In your course, select **Discussion**.
+#. In the **How to use edX discussions** graphic, select the **Receive
+   updates** check box.
```
Comment (debchatigny) [inline en_us/shared/students/sfd_discussions/keep_up.rst]: Change check box to checkbox.

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 4]
```diff
@@ -58,17 +58,54 @@ number.
...
 
 .. image:: ../../../shared/images/Discussion_ReceiveUpdates.png
   :width: 600
-  :alt: Discussion how to graphic with the Receive Updates check box circled
+  :alt: "How to use edX discussions" graphic with the Receive Updates check box
```
Comment (debchatigny) [inline en_us/shared/students/sfd_discussions/keep_up.rst]: Change check box to checkbox.

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 5]
```diff
@@ -58,17 +58,54 @@ number.
...
+================================
+Unsubscribing from Notifications
+================================
+
+If you want to unsubscribe from notifications, complete either of the following
+actions.
+
+* If you received an email message, select **Unsubscribe** in the message.
+* On the **Discussion** page in your course, clear the **Receive updates**
+  check box in the **How to use edX discussions** graphic.
```
Comment (debchatigny) [inline en_us/shared/students/sfd_discussions/keep_up.rst]: Change check box to checkbox.

**Comment by debchatigny** [comment]
Hi Sylvia,

A few minor comments inline; overall 👍

**Review by ssemenova — APPROVED** [review]

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 6]
```diff
@@ -58,17 +58,54 @@ number.
...
+
+================================
+Unsubscribing from Notifications
+================================
+
+If you want to unsubscribe from notifications, complete either of the following
+actions.
+
+* If you received an email message, select **Unsubscribe** in the message.
+* On the **Discussion** page in your course, clear the **Receive updates**
```
Comment (iloveagent57) [inline en_us/shared/students/sfd_discussions/keep_up.rst:112]: This will unsubscribe you from the daily digest, but it won't stop you from receiving first-response notifications to posts you are following.

File: en_us/shared/students/sfd_discussions/keep_up.rst [hunk 7]
```diff
@@ -58,17 +58,60 @@ number.
 For more information about voting for posts, see :ref:`Vote for Posts or
 Responses`.
 
+*****************************
```
Comment (srpearce) [inline en_us/shared/students/sfd_discussions/keep_up.rst]: Add anchor: Receiving Discussion Notifications
